cli/internal/plugin/logging.go

```diff
@@ -13,8 +13,10 @@ func jsonToLog(msg map[string]interface{}, l zerolog.Logger) {
 	case "info":
 		l.Debug().Fields(msg).Msg("")
 	case "warn":
+		p.warnings++
```

Comment:
@yevgenypats I'm not sure, but this (and `p.errors++`) looks like it may need to be made thread-safe with a mutex lock?

Reply:
@hermanschaaf  we are only accessing this in the thread that is reading the logs so I think it is fine

Reply:
Ah okay, got it 👍